geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/geoportal/CONST_vars.yaml

```diff
@@ -950,8 +950,7 @@ vars:
   content_security_policy_main_default_src_extra: ''
   content_security_policy_main_script_src_extra: '
     https://maps.google.com/
-    https://maps.googleapis.com/
-    https://www.google-analytics.com/
+    https://*.googleapis.com/
```

Comment:
Why not keep the www?

Reply:
Just copying what they had for Google Maps CSP, they apparently not put "map", that is not www that I removed :D

Reply:
Here: https://developers.google.com/tag-platform/tag-manager/csp?hl=fr#google_analytics_4_google_analytics?
There is no googleapis.com on script-src ...
Can you also add the previous URL in comment?

Reply:
You are confusing Analytics and Maps, both have their respective URLs: https://developers.google.com/maps/documentation/javascript/content-security-policy?hl=fr#allowlist_csp

And actually even maps.google should become *.google so I need to tweek thas as well.

Reply:
+1 Then one more reason to put the links as comments :-)

Reply:
Ok, I did add comments, is it okay for comprehension ?